src/locale/en.json

```diff
@@ -256,5 +256,11 @@
 	"areasOfWork": "areasOfWork",
 	"audience_undefined_label": "audience_undefined_label",
```

Comment:
from here onwards, these key-values shouldn't be here

Reply:
So... we need to move them? Or from now on?

Reply:
remove them